src/main/resources/messages.properties

```diff
@@ -344,7 +344,7 @@ sebserver.lmssetup.type.MOODLE_PLUGIN=Moodle with SEB Server Plugin
 sebserver.lmssetup.type.MOODLE_PLUGIN.tooltip=Moodle with SEB Server integration plugin installed
 sebserver.lmssetup.type.OPEN_EDX=Open edX
 sebserver.lmssetup.type.ANS_DELFT=Ans Delft
-sebserver.lmssetup.type.OPEN_OLAT=Open OLAT
```

Comment:
Same here, text name can be changed but the Enum name not because of legacy data